composer.json

```diff
@@ -36,6 +36,7 @@
         "yiisoft/yii-http": "^1.0"
     },
     "require-dev": {
+        "jetbrains/phpstorm-attributes": "^1.0",
```

Comment:
It doesn't seem to be dev dependency since it's used in `src`.

Reply:
Done